rollup.config.js

```diff
@@ -28,7 +28,7 @@ export default {
     }),
     commonjs({
       namedExports: {
-        'node_modules/lodash/lodash.js': [
+        'lodash/lodash.js': [
```

Comment:
This was an experiment I saw in a stack overflow thread and surprisingly made a difference.